wallets/lnc/client.js

```diff
@@ -141,3 +99,63 @@ function validateNarrowPerms (lnc) {
   // TODO: need to check for more narrow permissions
   // blocked by https://github.com/lightninglabs/lnc-web/issues/112
 }
+
+// default credential store can go fuck itself
+class LncCredentialStore {
```

Comment:
The default credential store had so many side effects it was incredibly difficult to troubleshoot. I started forking `lnc-web` (which we'll probably want to do eventually anyway), but simply removing the side effects was most of what we needed to make this work.

This also remove the password. As we discussed, we'll want to do one password to rule them all eventually, so there's no reason to try to support the buggy library one in the interim.